spec/acceptance/api/member_permissions_spec.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+resource "MemberPermission" do
+
+  let(:user) { Factory(:user, account: provider) }
+  let(:resource) { user.member_permissions }
+  let(:serialized) { representer.send(serialization_format, user: user) }
+
+  before do
+    provider.settings.allow_multiple_users!
+  end
+
+  shared_context "allowed sections are configured" do
+    before do
+      user.member_permissions.create(admin_section: :partners)
+    end
+  end
+
+  shared_context "allowed services are configured" do
+    before do
+      expect(user.account).to receive(:service_ids).and_return([1])
```

Comment:
Similarly to https://github.com/3scale/porta/pull/121/commits/673d25a9f107ac2ac4595b5985c942459fa47fd7#r237826336, it is better to send this `[1]` fetching the id dynamically, whatever it is. This way it will not conflict with other tests, plus it is easier to understand where this comes from 😄 
The same is applicable to all `1` in this file 😄